### ISSUE TITLE
loss NAN with C3STR

### ISSUE THREAD

**Issue by hazard7771** [ultralytics/ultralytics#1257]
### Search before asking

- [X] I have searched the YOLOv8 [issues](https://github.com/ultralytics/ultralytics/issues) and found no similar bug report.

### Bug

Hello! When l modified the structure of yolov8l, replacing Cf2 module with C3STR module. But when l am training the model, the loss is NAN 

### Are you willing to submit a PR?

- [ ] Yes I'd like to help by submitting a PR!

**Comment by AyushExel** [comment]
We haven't tested the C3STR module with the v8 architecture but it's possible that changing the network architecture requires changing(tuning) the hyper-parameters
